utils/copyTests.js

```diff
@@ -0,0 +1,13 @@
+import { fileURLToPath } from "url";
```

Comment:
This code seems to be copying over the tests for the CLI when it actually needs to be copying over the tests from the API project.

Reply:
Thank-you for the details.  In that case should I hardcode the urls & fetch it using getFilesFromRepo ?

Reply:
Done

Reply:
@zenweasel I am thinking we have to update Mongo URL here:- https://github.com/reactioncommerce/reaction/blob/trunk/apps/reaction/tests/util/setupJestTests.js